code/modules/modular_computers/file_system/programs/card.dm

```diff
@@ -200,14 +204,21 @@
 		if("PRG_age")
 			if(!computer || !authenticated_user || !target_id_card)
 				return TRUE
-			target_id_card.registered_age = params["id_age"]
+
+			var/new_age = params["id_age"]
+			if(!isnum(new_age))
+				stack_trace("[key_name(usr)] ([usr]) attempted to set invalid age \[[new_age]\] to [target_id_card]")
```

Comment:
Do we need to worry about `html_encode` here or does the VV window do that for us?

Reply:
![image](https://user-images.githubusercontent.com/24975989/110223962-88992500-7ecf-11eb-970b-df951cade72c.png)

Is that what you were asking?

Reply:
That works for me, yeah.